lib/lotus/routing/routes_inspector.rb

```diff
@@ -119,6 +148,10 @@ def to_s(formatter = FORMATTER, base_path = nil)
           end
         end
 
+        if header && !result.include?('Name'.freeze)
```

Comment:
Inline freeze is a 2.2+ feature, we're still supporting 2.1. Please add a constant.